src/main/java/com/amazon/janusgraph/graph/TravelGraphFactory.java

```diff
@@ -0,0 +1,19 @@
+package com.amazon.janusgraph.graph;
```

Comment:
perhaps you can merge the logic in this class also in the new test method you added in ScenarioTests